mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -364,35 +366,41 @@
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first parameter is the application name -->
     <string name="reduce_cookie_banner_details_panel_description_on_for_site_2">%1$s tries to automatically reject all cookie requests on supported sites.</string>
     <!-- Title text for the dialog use on the control branch of the experiment to determine which context users engaged the most -->
-    <string name="reduce_cookie_banner_control_experiment_dialog_title">Cookie banners begone!</string>
+    <string name="reduce_cookie_banner_control_experiment_dialog_title" moz:RemovedIn="112" tools:ignore="UnusedResources">Allow Firefox to reject cookie banners?</string>
```

Comment:
string changed so string ID should be updated.
Firefox should be a placeholder.
Comment should explain what placeholder will be replaced with

Reply:
Hi @Delphine,
in this case we are tagging the string to be removed in the future this string is not longer used anymore.
You can find more information about it [here](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/docs/Working-with-Strings.md#marking-an-unused-string-to-be-removed) :)

Reply:
oh, since I saw the actual string change underneath, I thought this might be an error

Reply:
Linter is complaining on our side now because of hardcoded name, but will figure this out on Monday (just a self note to self to take care of that)